deliveries/verifiable_offchain_computing-milestone_3.md

```diff
@@ -0,0 +1,45 @@
+We provide:
+- the worker, deployed on Phala Cloud, that compute the target number to found via a Verifiable Random Function and give the clue (LowerThan, GreaterThan) based on the number proposed by the user 
+- the ink! smart contract that manage the game (start a new game and save the guess made by the user) and consumes the response sent by the worker
+- a UI to display the data from the smart contract
+- the documentation to use these contracts
+
+
+**Deliverables**
+> Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which will be used for testing. If you don't provide a commit hash, we will work off the default branch of your repository. Thus, if you plan on continuing work after delivery, we suggest you create a separate branch for either the delivery or your continuing work. 
+
+| Number | Deliverable            | Link                                                                                                | Notes                                                                                                                                          |
+|--------|------------------------|-----------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1.     | ink! v6 smart contract | https://github.com/GuiGou12358/sc-rollup/tree/main/examples/guess-the-number/ink-v6-contract        |                                                                                                                                                | 
+| 2.     | Worker on Phala Cloud  | https://github.com/GuiGou12358/sc-rollup/tree/main/examples/guess-the-number/worker-phala-cloud     |                                                                                                                                                | 
+| 3.     | UI                     | https://github.com/GuiGou12358/guess-the-number-ui                                                  | A new, more sophisticated version, is under development but ideally we would need an graph indexer (SubQuery or Subsquid) for ink! v6 contract | 
+| 4.     | Documentation          | https://github.com/GuiGou12358/sc-rollup/tree/main/examples/guess-the-number                        |                                                                                                                                                | 
+| 5.     | Documentation          | https://medium.com/@guigoucrypto/verifiable-offchain-workers-for-ink-and-evm-contracts-d943f9022699 |                                                                                                                                                | 
```

Comment:
Nice article! Do you have an X account? Then either you or us could post about it and mention you.

Reply:
Thanks! The article was share here : https://x.com/ink_lang/status/1973779106467569940